### DISCUSSION TITLE
Toast overflows the screen on mobile view

### DISCUSSION THREAD

**Discussion by chaosdone** [twilio-labs/paste#3024] [Q&A] [answered]
Hi paste team, I'm Claudia Zhou from growth signup team. We are migrating our verification flow to paste and support both desktop and mobile. There's one issue we found with mobile view that the toast is overflowing most of width, like in screenshot. I wonder will there any plan to make toast adjustable or more mobile-compatible?

![image](https://user-images.githubusercontent.com/17061497/217348993-49298fcd-6a79-4b30-a4f9-d393e5d8dd70.png)

**Comment by loreina** [comment]
Hi @chaosdone,

We've added a ticket in our backlog to set the Toast width to 100% on smaller screen sizes. Someone on our team will follow up once that's been released. Thanks!

**Comment by loreina** [comment, marked as answer]
Hi @chaosdone, we've merged a fix for this and it'll be released in this PR https://github.com/twilio-labs/paste/pull/3095. Thanks!
